system-int-test/test/cmr/system_int_test/header_test.clj

```diff
@@ -1,10 +1,13 @@
 (ns cmr.system-int-test.header_test
   "Tests for headers in ingest and search responses"
   (:require [clojure.test :refer :all]
+            [clj-http.client :as client]
+            [clojure.string :as string]
             [cmr.system-int-test.utils.ingest-util :as ingest]
             [cmr.system-int-test.utils.search-util :as search]
             [cmr.system-int-test.utils.index-util :as index]
             [cmr.system-int-test.data2.collection :as dc]
+            [cmr.system-int-test.utils.url-helper :as url]
             [cmr.common-app.api.routes :as routes]))
```

Comment:
Nitpick: Could you line up the require block the same way as in common-app-lib/src/cmr/common_app/api/routes.clj? All our new or newly modified require block should line up that way.